kitchen/wazuh-puppet/Dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Does the container uses this? As it's declared as ARG before a FROM, I suspect it might not be recognized by the container. Please consider  using `ENV` or removing this ARG.

Reply:
Hi @jm404!

Thank you for reviewing.

I added this ARG in order to ignore the warning:
```
debconf: delaying package configuration, since apt-utils is not installed
```
Anyway the warning message does not disappear and there will be no need for this ARG definition.

I removed it.

Reference:
- https://github.com/phusion/baseimage-docker/issues/319#issuecomment-296431537

Kr,

Rshad